playpen/certs/extensions.txt

```diff
@@ -23,3 +23,7 @@ basicConstraints=CA:FALSE
 1.3.6.1.4.1.2312.9.2.0005.1.1=ASN1:UTF8:Pulp Production Fedora 16
```

Comment:
This file was accidentally included.  I can remove before the merge but maybe not worth the trouble since it's just playpen.